calculator.rb

```diff
@@ -0,0 +1,90 @@
+# Calculator (Dinner) - Loraine Kanervisto
+
+# Starts with user prompts and data collection
+
+#Collects first number
+puts "You find a calculator on the ground. You pick it up. The screen says:\n
+Enter a number>>"
+first_num = gets.chomp.to_f
+
+#Checks first number
+while first_num != "0" && first_num.to_i == 0
+  puts "That is totally not a number! Try agin!"
+  first_num = gets.chomp.to_f
+end
+
+#Collects math operation
+op_examples = "\nOPTIONS\n+, add, plus\n-, subtract, minus\n*, multiply, times\n/, divide, division\nexponent, to the power of\nmodulo"
+puts "\nNow choose a math operator! #{op_examples}"
```

Comment:
"Clarity over brevity," right? I think that's a Jeremy quote. I've updated the variable to meet your next few suggestions!